doc/classes/ParallaxLayer.xml

```diff
@@ -60,10 +61,13 @@
 	</methods>
 	<members>
 		<member name="motion_mirroring" type="Vector2" setter="set_mirroring" getter="get_mirroring">
+			 Mirroring of the [code]ParallaxLayer[/code], reapeat the [code]ParallaxLayer[/code]. Useful to create an infinite scrolling background. If an axis is set to 0 then that axis will have no mirroring. 
```

Comment:
The ParallaxLayer's [Texture] mirroring. Useful for creating an infinite scrolling background. If an axis is set to [code]0[/code] the [Texture] will not be mirrored. Default value: [code](0, 0)[/code].